cmd/test_load.go

```diff
@@ -202,3 +224,31 @@ func (lt *loadedTest) consolidateDeriveAndValidateConfig(
 
 	return nil
 }
+
+type syncWriter struct {
+	w io.Writer
+	m sync.Mutex
+}
+
+func (cw *syncWriter) Write(b []byte) (int, error) {
+	cw.m.Lock()
+	defer cw.m.Unlock()
+	return cw.w.Write(b)
+}
+
+func (lt *loadedTest) createFileForAppend(filename string) (io.WriteCloser, error) {
+	fs := lt.fileSystems["file"]
```

Comment:
I think this is a bug, you should use the `globalState.fs` that is passed to `loadAndConfigureTest()` here, not `lt.fileSystems`. Right now, `SSLKEYLOGFILE=./test.log k6 cloud script.js` will include `test.log` in the archive bundle

Reply:
fixed in [47c41d7](https://github.com/grafana/k6/pull/2487/commits/47c41d7977978c0330333854006dd0d2453ad7b2)